packages/scene-composer/src/augmentations/components/three-fiber/viewpoint/ViewCursorWidget.tsx

```diff
@@ -1,76 +1,94 @@
-import * as THREE from 'three';
-import React, { useContext, useEffect, useMemo, useRef } from 'react';
-import { extend, useLoader, useThree } from '@react-three/fiber';
-import { SVGLoader, SVGResult } from 'three-stdlib';
+import React, { useCallback, useContext, useMemo, useRef } from 'react';
+import { useFrame, useLoader, useThree } from '@react-three/fiber';
+import { AlwaysDepth, Box3, Color, DoubleSide, Group, Mesh, MeshBasicMaterial, Object3D, ShapeGeometry, Vector3 } from 'three';
```

Comment:
I'm going to make a non-blocking suggestion here:
I strongly advise that we use import * as THREE from 'three'; and I'll explain my reasoning as to why.

1. There are now possible name collisions. In our specific case 3 libraries including ours, have their own version of Vector3 (React-Three/Fiber, Ours in interfaces I believe, and Three)
2. The namespace is a ThreeJS standard. The author, all documentation and examples using ThreeJS will do the namespace style

Reply:
While I see where @jwills-jdubs is coming from, I respectfully disagree. `import * as Whatever from 'anylib';` is generally considered poor practice in Javascript, as it forces you to include entire libraries in the bundling process or relying on Tree Shaking to break things up.

The name collision issue can easily be worked around like:

```
import { AlwaysDepth as ThreeAlwaysDepth } from 'three';
```

Should never import more than you need.

Reply:
@TheEvilDev You good with this? I implemented the initial feedback since it made sense, but @jwills-jdubs makes a really good point about name collisions.

Reply:
So when this gets packaged we only take what we need from the lib? If we really don't take it all anyway it makes sense to me.

Reply:
Really good callouts on both sides guys. I'll convert to use @TheEvilDev's suggestion of `import { AlwaysDepth as ThreeAlwaysDepth } from 'three';`